funnel/views/proposal.py

```diff
@@ -152,6 +153,10 @@ def new_proposal(self):
             )  # Vote up your own proposal by default
             db.session.commit()
             flash(_("Your new session has been saved"), 'info')
+            for editor in proposal.project.active_editor_memberships:
+                proposal_submitted.send(
+                    proposal, project=proposal.project, editor=editor.user
+                )
```

Comment:
Send a signal just once. sender = proposal, project is additional data. This part of the code knows nothing about contacting editors. Process editors when receiving the signal.